src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public boolean getStatus() {
+        return isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getTaskType() {
+        return null;
+    }
+
+    public void printAddDetails(int taskCounter) {
```

Comment:
Consider refactoring the `printAddDetails()` function so that you don't need to duplicate part of the print statement in the child classes